integration_tests/mockData/prisonerByIdData.ts

```diff
@@ -80,6 +83,97 @@ const prisoners = {
         receptionDate: '2004-04-17',
         dateOfBirth: '1976-02-28',
         cellLocation: 'B-12-55',
+        restrictedPatient: false,
+      },
+    },
+  },
+  G9981UK: {
```

Comment:
I needed to add a few more stub prisoner data records in various prisons etc; and also set the `restrictedPatient` field for the existing prisoner records (forgot to do that in my previous PR)